examples/js/loaders/AMFLoader.js

```diff
@@ -48,18 +48,18 @@ THREE.AMFLoader.prototype = {
 
 	parse: function ( data ) {
 
-		function loadDocument( data ) {
+		function _loadDocument( data ) {
```

Comment:
> AMFLoader returns a Group of Object3Ds with children

Many loaders do that. Users can use `Object3D.traverse()` in the callback.

> I could always just create a default "null material"

Sounds reasonable.